subt_ign/launch/cloudsim_sim.ign

```diff
@@ -208,6 +226,12 @@
     $headless = true
   end
 
+  if !defined?(ros) || ros == nil || ros.empty?
```

Comment:
Consider adding a case for `ros == 'false' || ros == 'False'`.

Reply:
Fixed here: bb52583a7f4ef604606bedc115887c23d58d56d5